src/main/java/org/takes/rq/RqForm.java

```diff
@@ -341,6 +317,36 @@ public InputStream body() throws IOException {
         }
 
         /**
+         * Validate parameters.
+         * @param params Parameters
+         * @return Validated parameters
+         */
+        private static String[] validate(final String... params) {
```

Comment:
@xupyprmv Minor remark: it is preferable to use nouns instead of verbs if the method returns somehing, so `valid(params)` instead of `validate` (cf. http://www.yegor256.com/images/books/elegant-objects/seven-pages.pdf)